stdlib/builtins.pyi

```diff
@@ -849,8 +849,10 @@ class memoryview(Sequence[int]):
     def __setitem__(self, __s: slice, __o: ReadableBuffer) -> None: ...
     @overload
     def __setitem__(self, __i: SupportsIndex, __o: SupportsIndex) -> None: ...
-    if sys.version_info >= (3, 8):
-        def tobytes(self, order: Literal["C", "F", "A"] | None = ...) -> bytes: ...
+    if sys.version_info >= (3, 10):
+        def tobytes(self, order: Literal["C", "F", "A"] | None = "C") -> bytes: ...
+    elif sys.version_info >= (3, 8):
+        def tobytes(self, order: Literal["C", "F", "A"] | None = None) -> bytes: ...
```

Comment:
Looking at the code the default is also effectively `C` in 3.9, there just was no Argument Clinic yet to put the default: https://github.com/python/cpython/blob/c33aaa9d559398bbf2b80e891bf3ae6a716e4b8c/Objects/memoryobject.c#L2139.

Seems fine to keep as is though so we don't have to deal with stubtest allowlists.